src/main/cpp/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ set(CUDF_CPP_BUILD_DIR
   "$ENV{CUDF_CPP_BUILD_DIR}"
   CACHE STRING "path to libcudf build root"
 )
+set(CUDA_STATIC_RUNTIME ON)
```

Comment:
nit: Can we have some comments about this? It is a little odd that we have to set this two different ways.

Reply:
Added a comment.